pkg/apis/rbac/register.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rbac
+
+import (
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	"k8s.io/kubernetes/pkg/runtime"
+	"k8s.io/kubernetes/pkg/watch/versioned"
+)
+
+const GroupName = "rbac.authorization"
```

Comment:
From @lavalamp in another pull

> I don't think we did make an official judgement but I think it would be good to get in the habit of making group names DNS names.
https://github.com/kubernetes/kubernetes/pull/24902#discussion_r62391331

Let's go ahead and do that here.  That would be `rbac.authorization.k8s.io`.